repos/system_upgrade/el7toel8/workflows/inplace_upgrade.py

```diff
@@ -16,118 +16,158 @@ class IPUWorkflow(Workflow):
     description = '''No description has been provided for the InplaceUpgrade workflow.'''
 
     class FactsCollectionPhase(Phase):
+        '''Operating System Version Detection, IP, Hostname, RPMs, Services,
+        Augeas.
+        '''
         name = 'Facts collection'
         filter = TagFilter(FactsPhaseTag)
         policies = Policies(Policies.Errors.FailPhase,
                             Policies.Retry.Phase)
         flags = Flags()
 
     class ChecksPhase(Phase):
+        '''Inhibitors - Blockers, Shared Objects, Binaries, Languages,
+        Packages, Load Balance Support, User Management, Version Control
+        Systems, FreeRADIUS, IPA Bind Dyndb LDAP, SSSD, NTP, OpenLDAP, YPSERV,
+        Network, Databases, SELinux, Storage, System, HTTPD, OpenSSH, Postfix,
+        SQUID, DoveCot, Bind9 Configuration, RSYSLOG.
+        '''
```

Comment:
The description is vague. This is not only about inhibitors. Inhibitors are small part of actions that should happened here. Here should be put all user questions, should be checked what should be done and it should be planned. basically, it's subset of actions we usually do in PA during the check of the system. Just no actions should be done here.